src/components/SQForm/SQFormMultiValue.tsx

```diff
@@ -269,50 +253,51 @@ function SQFormMultiValue({
       'props' in endAdornment &&
       endAdornment.props
     ) {
-      const newEndAdornment = {
+      newEndAdornment = {
         ...endAdornment,
         props: {
           ...endAdornment.props,
-          className: `${endAdornment.props.className} ${clearButtonClasses.endAdornment}`,
+          style: {top: 'auto', bottom: '8px'},
```

Comment:
If I'm remembering correctly, the main driver behind this was that the little "x" icon to clear in the input field would do weird things if there were a lot of options and caused the field to be multiline. Coudl you confirm that if you add like 20 options and the input fields has to expand vertically that the x button is behaving properly.

Reply:
@20BBrown14 See loom for how it is currently behaving.  I believe this is what we want.

https://www.loom.com/share/c3133afc51b7438fb8f0c700ca1142d3